src/vivarium_conic_lsff/components/fortification/vitamin_a.py

```diff
@@ -86,3 +87,64 @@ def load_time_to_effect_data(builder: 'Builder') -> float:
         location = builder.configuration.input_data.location
         draw = builder.configuration.input_data.input_draw_number
         return sample_vitamin_a_time_to_effect(location, draw)
+
+
+class VitaminAFortificationEffect:
+
+    @property
+    def name(self) -> str:
+        return 'vitamin_a_fortification_effect'
+
+    def setup(self, builder: 'Builder'):
+        relative_risk_data = self.load_relative_risk_data(builder)
+        self.relative_risk = builder.lookup.build_table(relative_risk_data,
+                                                        parameter_columns=['age'])
+        paf_data = self.load_population_attributable_fraction_data(builder)
+        self.population_attributable_fraction = builder.lookup.build_table(paf_data,
+                                                                           parameter_columns=['age'])
+
+        self.effectively_covered = builder.value.get_value('vitamin_a_fortification.effectively_covered')
+        builder.value.register_value_modifier('vitamin_a_deficiency.exposure_parameters.paf',
+                                              self.population_attributable_fraction,
+                                              requires_columns=['age'])
+        builder.value.register_value_modifier('vitamin_a_deficiency.exposure_parameters',
+                                              self.adjust_vitamin_a_exposure_probability,
+                                              requires_columns=['age'])
+
+    def adjust_vitamin_a_exposure_probability(self, index: pd.Index, exposure_probability: pd.Series) -> pd.Series:
+        import pdb; pdb.set_trace()
+        effectively_covered = self.effectively_covered(index)
+        rr = self.relative_risk(index).lookup(index, effectively_covered)
+        return exposure_probability * rr
+
+    @staticmethod
+    def load_relative_risk_data(builder):
+        location = builder.configuration.input_data.location
+        draw = builder.configuration.input_data.input_draw_number
+        rr = sample_vitamin_a_relative_risk(location, draw)
+        relative_risk = pd.DataFrame({
+            'age_start': [0., 0.5],
+            'age_end': [0.5, 10.],
```

Comment:
Why is 10 used here and below?

Reply:
Avoids edge cases on the last time step.